.github/workflows/ci.yml

```diff
@@ -24,9 +24,12 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: dtolnay/rust-toolchain@stable
+      - run: cargo fmt --all -- --check
```

Comment:
Does this imply that non-`cargo fmt`'d PRs are rejected? If so, I would demote this to at most a warning.